src/main/java/com/linkedin/kafka/clients/largemessage/LargeMessage.java

```diff
@@ -17,15 +17,16 @@
  */
 public class LargeMessage {
   private final Map<Integer, ByteBuffer> _segments;
-  private final int _messageSize;
+  private final int _originalValueSize;
```

Comment:
And what's even more confusing (to me) is that occasionally the word "event" used.